### PR TITLE
fix-ID-12/feature/footer

### PR DESCRIPTION
**Pull request by kepkaklaudia**
In index.js i added disabled style to StyledVector as well, then i refactored the code. 
In styled.js i changed styles while we hover on Button and i added cursor not-allowed while disabled

### CODE REVIEW

File: src/common/Pagination/index.js [hunk 1]
```diff
@@ -6,26 +6,33 @@ export const Pagination = () => {
   const pageNumber = useSelector(selectPage);
   const totalPages = useSelector(selectTotalPages);
   const dispatch = useDispatch();
+  const firstPage = pageNumber === 1;
+  const lastPage = pageNumber === totalPages;
```
Comment (q270311) [inline src/common/Pagination/index.js:10]: good idea, maybe firstPage rename to isFirstPage (isLstPage)

**Review by q270311 — COMMENTED** [review]
good job :)

File: src/common/Pagination/index.js (hunk 1 above)
Comment (kepkaklaudia) [reply src/common/Pagination/index.js:10]: You're right, that would be better!

**Review by panijoanna — APPROVED** [review]
All consistent.

**Pull request merged**